alpine/src/main/java/io/delta/alpine/actions/AddFile.java

```diff
@@ -34,7 +35,7 @@ public URI getPathAsUri() throws URISyntaxException {
     }
 
     public Map<String, String> getPartitionValues() {
-        return partitionValues;
+        return Collections.unmodifiableMap(partitionValues);
```

Comment:
Return an immutable map so that the caller cannot change the content of `partitionValues`.